docs/_running-tests/from-local-system.md

```diff
@@ -14,6 +14,13 @@ On Windows, be sure to add the Python directory (`c:\python2x`, by default) to
 your `%Path%` [Environment Variable](http://www.computerhope.com/issues/ch000549.htm),
 and read the [Windows Notes](#windows-notes) section below.
 
+Install `virtualenv` using the following commands:
+
+```bash
+easy_install pip
```

Comment:
`pip` should be installed by default with at all recent Python versions (from 2.7.9, from Dec 2014)?